opentelemetry-sdk/tests/error_handler/test_error_handler.py

```diff
@@ -25,7 +25,8 @@
 
 
 class TestErrorHandler(TestCase):
-    def test_default_error_handler(self):
+    @patch("opentelemetry.sdk.error_handler.iter_entry_points")
```

Comment:
What's this change for?

Reply:
when running the test locally with the `eachdist` script this test fails. The problem is that there are some entry points already setup when installing a dev environment with `eachdist develop`. This test however checkes the default error handler when no error handler entry points are configured.